### ISSUE TITLE
Vulnerability testing for OP-TEE

### ISSUE THREAD

**Issue by jigi-kim** [OP-TEE/optee_os#2231]
Hi all.

Recently I got an interest on this project, and now I have a short question about OP-TEE (or generally, every TEEs). How can someone say that a TEE is, practically, not vulnerable? or/and If someone forked and modified OP-TEE source codes, how can he/she can say that the modified version is still safety as original OP-TEE does? Is there any vulnerability test suites, especially open-sourced one, or way to prove it?

**Comment by jenswi-linaro** [comment]
We don't have any test suite to test against general vulnerabilities. We have https://github.com/OP-TEE/optee_test, but that's mainly focused on APIs.

**Comment by jenswi-linaro** [comment]
No, the certification is also mostly focused on APIs. If it's trustworthy or not is a decision for the one integrating it in a SoC. In this project we try to make this trustworthy, but we can't decide what you should trust or not.
